### PR TITLE
Use main as live branch for github action

### PR DESCRIPTION
**Pull request by codezz**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,7 +23,8 @@
   },
   "scripts": {
     "start": "react-scripts start",
-    "build": "react-scripts build",
+    "build": "react-scripts build && npm run build:clean",
+    "build:clean": "cd build && cp static/js/main*.js main.js && cp static/css/main*.css main.css",
```
Comment (bodyionita) [inline package.json:27]: I have but one question. If the clean line removes the giberish form the filenames, who changes the main.css and .js in the index.html?

File: wp/core/Plugin.php [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+	}
+
+	public function init_actions() {
+		add_shortcode( 'stakeborg_badges', array( $this, 'badges_shortcode' ) );
+	}
+
+	public function enqueue_assets() {
+		wp_register_style( 'stakeborg-badges', SBORG_BADGES_ASSETS_URL . 'build/main.css', array(), SBORG_BADGES_VERSION, 'all' );
+		wp_register_script( 'stakeborg-badges', SBORG_BADGES_ASSETS_URL . 'build/main.js', array(), SBORG_BADGES_VERSION, true );
```
Comment (bodyionita) [inline wp/core/Plugin.php:36]: is it this that registers the 2 files? and thus gets them loaded?

**Review by bodyionita — COMMENTED** [review]
I just have the question regarding the .css and .js imports, but if those are fine I can approve this, i'm happy with the rest

**Review by bodyionita — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (codezz) [reply package.json:27]: I just added a new script that copies the generated main js and css files but without the hash in the filename. this happens each time you do npm run build

File: wp/core/Plugin.php (hunk 2 above)
Comment (codezz) [reply wp/core/Plugin.php:36]: In WP you can register scripts and styles(using wp_register_style/script) but not actually loading them in the page. You can later use the handle used to register the scripts to actually enqueue them. This flow is more of a best practice that allows other devs to use the WP filters to dequeue maybe your script. 
https://developer.wordpress.org/reference/functions/wp_register_script/

File: package.json (hunk 1 above)
Comment (bodyionita) [reply package.json:27]: that i figured

File: wp/core/Plugin.php (hunk 2 above)
Comment (bodyionita) [reply wp/core/Plugin.php:36]: and yes, i meant registering them globally or whatever like a singleton. thanks :)

**Pull request merged**